clients.json

```
@@ -806,13 +806,23 @@
     "active": true
   },
 
+  {
+    "name": "handy-redis",
+    "language": "Node.js",
+    "repository": "https://github.com/mmkal/handy-redis",
+    "description": "A wrapper around node_redis with Promise and TypeScript support.",
+    "authors": ["mmkal"],
```

This is supposed to be your (the author's) Twitter handle. I can see one exists, but it appears inactive so if you're not using it you can just leave it empty, i.e.:
```suggestion
    "authors": [],
```

---

Thanks for pointing out! I don't have that twitter handle sadly. I don't have a GitHub-equivalent twitter right now. Will open a new PR someday when I do 🙃